fda-model/compute/ComputeCompleteMessage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace compute
+{
+    public class ComputeCompleteMessage : HEC.MVVMFramework.Base.Interfaces.IMessage
```

Comment:
Seems a little odd to make a whole class with a hardcoded message like this. Why not use a more generic class and inject the message in the constructor? I'd think Framework would have something out of the box for that.